src/riscv-hybrid-integration.adoc

```diff
@@ -324,7 +324,7 @@ it is not possible to disable CHERI checks completely.
 endif::[]
 
 {cheri_default_ext_name} includes functions to disable explicit access to CHERI
-registers.  The following occurs when executing code in a privilege mode that
+registers and instructions.  The following occurs when executing code in a privilege mode that
```

Comment:
I thought this was implicit because any CHERI instruction that uses a CHERI register is not possible, but happy to add this clarification.

Reply:
@arichardson I agree, it is implicit. For some CSRs, the text explicitly says "registers and instructions" (https://github.com/riscv/riscv-cheri/blob/main/src/riscv-hybrid-integration.adoc?plain=1#L446-L448) whereas for others the text only refers to registers. We might want to have that consistent. I am fine with either way.

If we go with only referring to registers, we might want to make it clear somewhere that this implies that code cannot execute CHERI instructions.

Reply:
pushing to this branch resulted in a new PR (no idea why) but here it is: https://github.com/riscv/riscv-cheri/pull/474